cmd/root.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"encoding/json"
+	"net"
+	"net/http"
+
+	"github.com/anmitsu/go-shlex"
+	"github.com/spf13/cobra"
+)
+
+type toCheck struct {
+	Name      string `json:"name"`
+	Type      string `json:"type"`
+	Value	  string `json:"value"`
+}
```

Comment:
@fabriciojs thinking about moving everything here to a separete "checks.go" file/class so the command itself is a more clean and lean file